ansible/sandbox.yml

```diff
@@ -0,0 +1,103 @@
+---
+- hosts: all
+  gather_facts: no
+
+  tasks:
+  - name: install python 2
+    raw: test -e /usr/bin/python || (apt -y update && apt install -y python2.7)
+    become: yes
+
+- name: Provision ISIC Archive
+  hosts: sandbox
+
+  vars:
+    # TODO figure out how to programatically detect this:
+    # https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/nvme-ebs-volumes.html
+    volume_path: "/dev/nvme1n1"
```

Comment:
This is wrong on the provisioned machine. There, `/dev/nvme1n1` is actually the root volume, and `/dev/nvme0n1` is the storage volume. I've manually edited `/etc/fstab` on the machine for now, but we should figure this out.